lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-lenovo-physical_infra_manager-physical_server.rb.rb

```diff
@@ -0,0 +1,4 @@
+module MiqAeMethodService
+  class MiqAeServiceManageIQ_Providers_Lenovo_PhysicalInfraManager_PhysicalServer < MiqAeServicePhysicalServer
+  end
```

Comment:
If this method is not exposing any associations or methods it is not required.  The engine will auto-generate it when the class is first referenced.